src/current/v23.1/cpu-profiler.md

```diff
@@ -0,0 +1,71 @@
+---
+title: CPU Profiler
+summary: Verify that your cluster's data replication, data placement, and zone configurations are working as expected.
+keywords: cpu profile, cpu profiles, cpu profiler
+toc: true
+docs_area: manage
+---
+
+{% include_cached new-in.html version="v23.1" %} The CPU Profiler automatically captures CPU profiles. A CPU profile shows the functions that use the most CPU time, sampled over a window of time.
+
+Use CPU Profiles to troubleshoot spikes in CPU utilization or erratic CPU load on certain nodes. You can collect a CPU Profile manually on the [Advanced Debug page](ui-debug-pages.html). However, it may be difficult to manually capture a profile of a short CPU spike at the right point in time. Automatic CPU profile capture enables the investigation of CPU load in this and other cases, such as periodic high CPU.
+
+## Configuration
+
+You can configure automatic CPU profile capture with the following [cluster settings](cluster-settings.html):
+
+Cluster Setting | Description | Default Value | Recommended Value
+----------------|-------------|---------------|------------------
+`server.cpu_profile.enabled` | The ON/OFF switch of the CPU profiler | `false` | 
+`server.cpu_profile.cpu_usage_combined_threshold` | The baseline value for when CPU profiles should be taken. Collect profiles from each node that meets threshold. | MAX integer, such as `9223372036854775807` | `80`
+`server.cpu_profile.interval` | The period of time after which the high-water mark resets to the baseline value | `5m0s` (5 minutes) | `1m40s` (100 seconds)
+`server.cpu_profile.duration` | The length of time a CPU profile is taken | `10s` (10 seconds) | `1s` or `2s`
+`server.cpu_profile.total_dump_size_limit` | Maximum combined disk size for preserving CPU profiles | `128 MiB` (128 Mebibytes) |
+
+### Enabling automatic CPU profile capture
+
+- Set `server.cpu_profile.enabled` to `true`
+- Set `server.cpu_profile.cpu_usage_combined_threshold` to a value between `0` and `100`.
+
+{{site.data.alerts.callout_info}}
+Only setting `server.cpu_profile.enabled` to `true` will not generate a CPU profile.
+{{site.data.alerts.end}}
+
+### Recommended values
+
+- Set `server.cpu_profile.cpu_usage_combined_threshold` to `80` for 80%.
+- Set `server.cpu_profile.duration` to a lower value, for example `1s` or `2s`. This reduces [overhead](#overhead).
```

Comment:
Suggestion to tweak the language to say something along the lines of: "This minimizes the impact of overhead on your cluster compared to the current default values"

^ Reasoning is that the current statement "This reduces overhead" makes it sound like enabling this will reduce overhead on your cluster. That's not true. What's more accurate is that these lower values will result in **less** overhead than what would have resulted from the default cluster setting values

Reply:
Modified to say "This minimizes the impact of [overhead](#overhead) on your cluster compared to the current default value."